client/post-editor/post-editor.jsx

```diff
@@ -1396,26 +1395,21 @@ const enhance = flow(
 				nestedSidebarTarget: getNestedSidebarTarget( state ),
 			};
 		},
-		dispatch => {
-			return bindActionCreators(
-				{
-					setEditorLastDraft,
-					resetEditorLastDraft,
-					receivePost,
-					editPost,
-					savePostSuccess,
-					setEditorModePreference: savePreference.bind( null, 'editor-mode' ),
-					setEditorSidebar: savePreference.bind( null, 'editor-sidebar' ),
-					setLayoutFocus,
-					setNextLayoutFocus,
-					saveConfirmationSidebarPreference,
-					recordTracksEvent,
-					setNestedSidebar,
-					closeEditorSidebar,
-					openEditorSidebar,
-				},
-				dispatch
-			);
+		{
+			setEditorLastDraft,
+			resetEditorLastDraft,
+			receivePost,
+			editPost,
+			savePostSuccess,
+			setEditorModePreference: partial( savePreference, 'editor-mode' ),
+			setEditorSidebar: partial( savePreference, 'editor-sidebar' ),
```

Comment:
When I tested the PR I noticed this doesn't work (the sidebar prefs are never saved).

Though, it's not this PR's fault: simply put, `setEditorSidebar` is never used.
Apparently https://github.com/Automattic/wp-calypso/pull/19294 removed it:
- [Here](https://github.com/Automattic/wp-calypso/pull/19294/files#diff-914bdeefa512de6f78c9b7988546d6e3L274) we have it.
- [Here](https://github.com/Automattic/wp-calypso/pull/19294/files#diff-d57de61cfe0b79c7540ff6f439917184R25), which is the function that replaced the close sidebar part, not anymore.

Same goes for the open sidebar part.

We might want to fix this in a follow up.

Reply:
@Copons,  Closing the loop on this. It should be addressed in: #20379

Thanks for leaving this comment, it helped me track down this bug really quickly!!